datus/tools/bi_tools/superset_adaptor.py

```
@@ -0,0 +1,316 @@
+            self._ensure_authenticated()
+            if self._auth_header_value:
+                if self.auth_type == AuthType.LOGIN:
+                    headers.setdefault("Authorization", self._auth_header_value)
+                else:
+                    # For test
+                    headers.setdefault("x-csrftoken", self._auth_header_value)
+
+        try:
+            response = self._client.request(method, url, headers=headers, **kwargs)
+            response.raise_for_status()
+            return response
```

Ruff (0.14.8)

Consider moving this statement to an `else` block

(TRY300)